README.md

```diff
@@ -23,8 +23,9 @@
 
 ## Start server
 
-`rails s`
+`foreman start`
 
 ## Google Auth
 
-http://localhost:3000/auth/google_oauth2/callback
+- Login: http://localhost:3000/auth/google_oauth2
```

Comment:
et ça se passe comment en prod ?

Reply:
C'est https://octo-moss-back.herokuapp.com/auth/google_oauth2.
Je me suis noté de faire un README tout propre et explicite dans une autre PR. ;)

Reply:
ça marche 👍